CHANGELOG.md

```diff
@@ -1,3 +1,145 @@
+# 0.1 - Jan XXX, 2025 - XXX
+
+## API Updates
+ * The `lightning-liquidity` crate has been moved into the `rust-lightning`
+   git tree, enabling support for both sides of the LSPS channel open
+   negotiation protocols (#3436).
+ * This release includes support for BIP 353 Human Readable Names resolution.
+   With the `dnssec` feature enabled, simply call
+   `ChannelManager::pay_for_offer_from_human_readable_name` with a list of
+   lightnig nodes which have the `dns_resolver` feature flag set (e.g. those
+   running LDK with the new `lightning-dns-resolver::OMDomainResolver` set up
+   to resolve DNS queries for others) and a Human Readable Name (#3346, #3179, #3283).
+ * Asynchronous `ChannelMonitorUpdate` persistence (i.e. the use of
+   `ChannelMonitorUpdateStatus::InProgress`) is now considered beta-quality.
+   There are no known issues with it, though the likelihood of unknown issues
+   is high (#3414).
+ * `ChannelManager`'s `send_payment_with_route` and `send_spontaneous_payment`
+   were removed. Use `send_payment` and `send_spontaneous_payment_with_retry`
+   (now renamed `send_spontaneous_payment`) instead (#3430).
+ * `ChannelMonitor`s no longer need to be re-persisted after deserializing the
+   `ChannelManager` before beginning normal operation. As such,
```

Comment:
I feel like the note in backwards compatibility should be referenced here. IIUC, persistence is required at least once? And maybe the following sentence should come first to emphasize the API change here.